tensorflow/core/ops/array_ops.cc

```diff
@@ -410,6 +410,20 @@ axis: Dimension along which to unpack.  Negative values wrap around, so the
 output: The list of tensors unpacked from `value`.
 )doc");
 
+REGISTER_OP("BroadcastTo")
+    .Input("input: T")
+    .Input("shape: int32")
```

Comment:
Might be worth it to support int64 shapes as well

Reply:
Thanks @alextp. Done.